e2e-tests/playwright/support/ui/components/channels/sidebar_right.ts

```diff
@@ -3,16 +3,49 @@
 
 import {expect, Locator} from '@playwright/test';
 
+import {components} from '@e2e-support/ui/components';
+
 export default class ChannelsSidebarRight {
     readonly container: Locator;
 
+    readonly input;
+    readonly sendButton: Locator;
+
     constructor(container: Locator) {
         this.container = container;
+
+        this.input = container.getByTestId('reply_textbox');
+        this.sendButton = container.getByTestId('SendMessageButton');
     }
 
     async toBeVisible() {
         await expect(this.container).toBeVisible();
     }
+
+    async postMessage(message: string) {
+        await this.input.fill(message);
+    }
+
+    async sendMessage() {
+        await this.sendButton.click();
+    }
+
+    /**
+     * Returns the value of the textbox in RHS
+     */
+    async getTextboxValue() {
```

Comment:
For consistency with element name.
```suggestion
    async getInputValue() {
```

Reply:
Done ✅